smart-village-solutions/smart-village-app-app
# chore: update react native elements

- added a style change for the height of the `Input` as I read in the update changes so that the inputs have the same design as before the update
- edited because as I read in the update notes of the `rn-elements` package, the `height`, `width` and `borderRadius` props for the `Overlay` component have been added to the `overlayStyle`
- fixed text error in the title of the `Button` in `NoticeboardCreateForm`
- added new prop instead of deleted `ListItem` props in `rn-elements` package
- updated `style` prop in `Image` component to `childrenContainerStyle`
- added `accessibilityRole` prop to prevent application crash due to `Checkbox` component of `rn-elements` package
- added package `react-native-status-bar-height` deleted from package `rn-elements` to `package.json`

SVA-888


## How to test: 
please navigate through the app and report any UI bugs you see. The biggest changes have been made to pages with lists.



## Screenshots:

|before (textliste)|after (textliste)|
|--|--|
![IMG_1982](https://user-images.githubusercontent.com/11755668/212946383-980153d3-ce74-48a4-bb98-245788e0df68.PNG) | ![IMG_1988](https://user-images.githubusercontent.com/11755668/213150456-65f7f204-79f6-4d67-8edd-8fbf6703d366.PNG)
|before (list with small pictures)|after (list with small pictures)|
![IMG_1983](https://user-images.githubusercontent.com/11755668/212946579-9ee6a4c1-218d-490e-8315-34d15fe6abe9.PNG) | ![IMG_1989](https://user-images.githubusercontent.com/11755668/213150659-b7f8d445-ae95-48b4-ae85-40bfd41a1d84.PNG)
|before (list with large images)|after (list with large images)|
![IMG_1984](https://user-images.githubusercontent.com/11755668/212946807-347338df-3458-4f0e-a55e-a87bb778d46d.PNG) | ![IMG_1990](https://user-images.githubusercontent.com/11755668/213152478-42c25daf-8511-481a-bf84-402096e28d75.PNG)
